packages/scandipwa/src/component/ProductPrice/ProductPrice.style.scss

```diff
@@ -41,18 +41,37 @@
     del {
         font-size: 14px;
         font-weight: 500;
-        opacity: .5;
+        opacity: .48;
         display: inline-block;
         margin-inline-end: 12px;
     }
 
     &-HighPrice {
         color: var(--price-discount-color);
         line-height: 18px;
+
+        @include desktop {
+            line-height: 14px;
+        }
     }
 
     &-PriceValue {
         margin-block-start: 6px;
+
+        @include desktop {
+            line-height: 28px;
+        }
+
+        @include mobile {
+            color: var(--h2-color-mobile);
+            font-size: var(--h2-font-size-mobile);
+            font-style: var(--h2-font-style-mobile);
+            font-weight: var(--h2-font-weight-mobile);
+            line-height: var(--h2-line-height-mobile);
+            margin-block-end: 21px;
+            margin-block-start: 25px;
+            text-transform: var(--h2-text-transform-mobile);
```

Comment:
Wait, why it should be same as h2?

Reply:
Wont it be like if client will set h2 to be 20px on mobile, so that price value will be also 20px ?